arrow-fx/src/test/kotlin/arrow/fx/ScheduleTest.kt

```diff
@@ -179,23 +179,15 @@ class ScheduleTest : UnitSpec() {
     }
 
     "Schedule.recurs(n: Int)" {
-      forAll(Gen.intSmall().filter { it < 1000 && it != 0 }) { i ->
-        val res = Schedule.recurs<ForId, Int>(Id.monad(), i).runIdSchedule(0, i + 1)
+      forAll(Gen.intSmall().filter { it < 1000}) { i ->
+        val res = Schedule.recurs<ForId, Int>(Id.monad(), i).runIdSchedule(0, max(1, i + 1))
```

Comment:
```suggestion
        val res = Schedule.recurs<ForId, Int>(Id.monad(), i).runIdSchedule(0, max(2, i + 1))
```

Reply:
This is not the solution. The test assumption below is wrong. It needs to be changed to check if the last decision is false and all the prior ones are the true and have the correct result

Reply:
And it needs to do that in any case, even if `n <= 0`